docs/cases/indicators-of-compromise.asciidoc

```diff
@@ -120,7 +120,13 @@ image::images/indicator-added-to-case.png[An indicator attached to a case]
 
 When you attach an indicator to a case, the indicator is added as a new comment with the following details:
 
-* *Indicator name*: Click the linked name to open the Indicator details flyout, which contains additional information about the indicator. Indicator details are in JSON format.
+* *Indicator name*: Click the linked name to open the Indicator details flyout, which contains the following informational tabs:
```

Comment:
```suggestion
* *Indicator name*: Click the linked name to open the Indicator details flyout, which contains the following tabs:
```
Nit, but informational seems like an extra word that might not be maximally friendly to non-native speakers. I think saying it's a details flyout and describing the tabs makes it unnecessary